frontend/mulighetsrommet-veileder-flate/cypress/e2e/mulighetsrommet.cy.js

```diff
@@ -19,35 +23,55 @@ describe('Tiltaksgjennomføringsoversikt', () => {
     cy.checkPageA11y();
   });
 
+  it('Sjekk om bruker har innsatsgruppe eller servicegruppe', () => {
+    const path = 'nokkel';
```

Comment:
Hva gjør denne?

Reply:
Sjekker at bruker initielt har innsatsgruppe ved at "nokkel" står i URL. Quick-fix, men fikk ikke laget tester mot nettverket, så det funker for now